src/ext/java/org/opentripplanner/ext/emissions/EmissionsFilter.java

```diff
@@ -43,10 +44,13 @@ public List<Itinerary> filter(List<Itinerary> itineraries) {
         .filter(leg -> leg.getMode() == TraverseMode.CAR)
         .toList();
 
-      calculateCo2EmissionsForCar(carLegs)
-        .ifPresent(co2 -> {
-          itinerary.setEmissionsPerPerson(new Emissions(co2));
-        });
+      Optional<Grams> co2ForCar = calculateCo2EmissionsForCar(carLegs);
+
+      if (co2ForTransit.isPresent() && co2ForCar.isPresent()) {
+        itinerary.setEmissionsPerPerson(new Emissions(co2ForTransit.get().plus(co2ForCar.get())));
+      } else if (co2ForTransit.isPresent()) {
+        itinerary.setEmissionsPerPerson(new Emissions(co2ForTransit.get()));
+      } else co2ForCar.ifPresent(grams -> itinerary.setEmissionsPerPerson(new Emissions(grams)));
```

Comment:
Wrap this with `{}` to improve readability.

Reply:
Maybe change this to be else if with (co2ForCar.isPresent()) and then you can use the co2ForCar.get() inside it.

Reply:
I originally had written it that way but intellij suggested to change it like this. However, it is fine with the {} wrapping. Should I just wrap it or change it entirely despite the intellij suggestion?